src/main/java/org/opensearch/knn/index/query/KNNWeight.java

```diff
@@ -241,20 +240,20 @@ private String buildDiskBasedSearchExplanation() {
         return sb.toString();
     }
 
-    private KNNScorer getOrCreateKnnScorer(LeafReaderContext context) throws IOException {
+    private Scorer getOrCreateKnnScorer(LeafReaderContext context) throws IOException {
```

Comment:
Out of curiousity, why change here?

Reply:
In future if we change scorer, we don't have to make change to this method. And i don't see need for this to KNNScorer explicitly, hence changed it to Scorer. WDYT?